NCS.DSS.Interaction/GetInteractionHttpTrigger/Function/GetInteractionHttpTrigger.cs

```diff
@@ -38,34 +38,57 @@ public GetInteractionHttpTrigger(IResourceHelper resourceHelper, IHttpRequestHel
         [Display(Name = "Put", Description = "Ability to return all interactions for a given customer.")]
         public async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "Customers/{customerId}/Interactions/")] HttpRequest req, string customerId)
         {
+            _logger.LogInformation("Function {FunctionName} has been invoked", nameof(GetInteractionHttpTrigger));
+
             var touchpointId = _httpRequestMessageHelper.GetDssTouchpointId(req);
             if (string.IsNullOrEmpty(touchpointId))
             {
-                log.LogInformation("Unable to locate 'APIM-TouchpointId' in request header.");
+                _logger.LogInformation("Unable to locate 'TouchpointId' in request header.");
                 return new BadRequestObjectResult(HttpStatusCode.BadRequest);
             }
 
-            log.LogInformation("Get Interaction C# HTTP trigger function  processed a request. By Touchpoint. " + touchpointId);
-
             if (!Guid.TryParse(customerId, out var customerGuid))
+            {
+                _logger.LogWarning("Unable to parse 'customerId' to a GUID. Customer GUID: {CustomerID}", customerId);
                 return new BadRequestObjectResult(customerGuid);
+            }
```

Comment:
Need a log stating that input validation is complete, see https://github.com/SkillsFundingAgency/dss-transfer/pull/31/files#diff-695bbf80412b14a3c3dcfde0ff6fd17ff9d7119b08dcd1741520ff6e7ee8cd9f line 71 for an example